rotkehlchen/chain/ethereum/defi/zerionsdk.py

```diff
@@ -341,7 +339,7 @@ def handle_protocols(
             return None
 
         return DefiBalance(
-            token_address=to_checksum_address(token_address),
+            token_address=deserialize_ethereum_address(token_address),
```

Comment:
same as mentioned above. Add at docstring. But since it's called from `get_single_balance()` you can just let it go through there and have it caught at the same place I mentioned in the other comment.